src/Core/AuthenticationHelpers/ClientRoleHeaderAuthenticationMiddleware.cs

```diff
@@ -108,12 +108,14 @@ public async Task InvokeAsync(HttpContext httpContext)
             // only in the non-hosted scenario.
             if (!_isLateConfigured)
             {
-                string requestAuthStatus = isAuthenticatedRequest ? AuthorizationType.Authenticated.ToString() :
-                    AuthorizationType.Anonymous.ToString();
-                _logger.LogDebug($"{HttpContextExtensions.GetLoggerCorrelationId(httpContext)}" +
-                    $"Request authentication state: {requestAuthStatus}.");
-                _logger.LogDebug($"{HttpContextExtensions.GetLoggerCorrelationId(httpContext)}" +
-                    $"The request will be executed in the context of {clientDefinedRole} role");
+                string requestAuthStatus = isAuthenticatedRequest ? AuthorizationType.Authenticated.ToString() : AuthorizationType.Anonymous.ToString();
+                _logger.LogDebug(
+                    message: "{correlationId} Request authentication state: {requestAuthStatus}.",
+                    HttpContextExtensions.GetLoggerCorrelationId(httpContext),
```

Comment:
nit: Introduce a variable for `correlationId` to use it rather than making the call to the extension twice.

Reply:
updated